proxy.go

```diff
@@ -240,11 +276,25 @@ func (p *Proxy) HTTPServe() {
 // the latest data.
 func (p *Proxy) RefreshDestinations(serviceName string, ring *consistent.Consistent, mtx *sync.Mutex) {
 
+	log.WithFields(logrus.Fields{
+		"discovererType": reflect.TypeOf(p.Discoverer),
+		"serviceName":    serviceName,
+	}).Info("Refreshing destinations")
 	start := time.Now()
 	destinations, err := p.Discoverer.GetDestinationsForService(serviceName)
+
+	log.WithFields(logrus.Fields{
+		"destinations": destinations,
+		"service":      serviceName,
+	}).Info("Got destinations")
```

Comment:
I think this should be Debug lest we create a stream of info logs that don't tell us a lot?